tests/modules/encounters/test_models.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name,missing-docstring
+import json, logging
+
+
+def test_encounter_add_owner(db):
+
+    from app.modules.users.models import User
+    from app.modules.encounters.models import Encounter
+
+    test_user = User(
```

Comment:
Better to use the utils.generate_user_instance() function

Reply:
Great! Now I know about that, thanks.

Reply:
I agree with @hwindsor, we should be using utility functions like this as much as we can